### ISSUE TITLE
hauppauge remote no longer working after new kernel

### ISSUE THREAD

**Issue by pootler123** [OpenELEC/OpenELEC.tv#3610]
Since dev build 1008 by milhouse, my hauppauge remote has stopped being recognized.
That build has the new kernel ( 3.17)
every subsequent build fails with this remote including the latest beta official build.
Popcornmix  and milhouse suggested providing 'lsmod' for working and non working builds.
Still no luck yet  so I  suggested raising a ticket
The remote is a  HAUPPAUGE WinTV TV REMOTE A415-HPG-WE-A and is used with hauppauge wintv duet usb dvb tuner.
The tuner still works, but the remote does not.
enclosed are the lsmods for working and non-working builds.
Although not very technical, I may be able to provide extra info if required.
thanks

http://www.pastebin.ca/2871787

**Comment by mrdominuzq** [comment]
still an issue ?

**Comment by pootler123** [comment]
HI,
 to be quite honest gave up on this months ago as nobody seemed interested to help!
Bought another cheap remote with receiver- seems to work fine.

I also am now using two geniatech t230  dvbt2 tuners instead of the hauppauge, very good and good HD pics
Cheers ;-)
pootler
p.s anybody interested in two hauppauge usb tuners? 


     On Monday, 13 April 2015, 20:06, Patrick Rasmussen <notifications@github.com> wrote:
   

 still an issue ?—

**Comment by mrdominuzq** [comment]
well everybody here is interested to help, but we also have daily life to take care of :)

**Comment by pootler123** [comment]
HI Patrick,
My reply wasn't meant to be at all ungrateful!
I understand we all have a life :-)
It is partially because of that , that I realise not every problem raised is going to be ''intriguing'' enough to receive people's priority- hence I gave it a good while and then decided to move on.
Because I have  changed usb tuners as well, the problem has gone away for me, and -  as mentioned  - there did not seem to be anyone else who was experiencing the same problem, so I decided to call it a day on that one!
Cheers
pootler
 


     On Tuesday, 14 April 2015, 19:29, Patrick Rasmussen <notifications@github.com> wrote:
   

 well everybody here is interested to help, but we also have daily life to take care of :)—
